frontend/src/app/forms/steps/submit-page/submit-page.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-submit-page',
+  templateUrl: './submit-page.component.html',
+  styleUrls: ['./submit-page.component.scss'],
+})
+export class SubmitPageComponent {
+  constructor(private router: Router) {}
+
+  onStartApplication(): void {
```

Comment:
What's the need for action binding in this component? Just remove the method. As of now there is no interaction in this page. It is just a static content.